processors/base/helpers.py

```diff
@@ -58,6 +58,16 @@ def default(self, obj):
         return json.JSONEncoder.default(self, obj)
 
 
+def clean_list(raw_list):
+    """Remove falsy values from list.
+    """
+    cleaned_list = []
+    for value in raw_list:
+        if value:
+            cleaned_list.append(value)
+    return cleaned_list
```

Comment:
This is programming style and you're certainly aware, but just if you missed it, this could be rewritten as:

```python
def clean_list(raw_list):
    return [value for value in raw_list
            if value]
```

Which seems more pythonic to me.